networks/80002/markets.json

```diff
@@ -1,9 +1,4 @@
 [
-    "spot://DUCKIES/USD",
-    "spot://USDT/USD",
-    "spot://MATIC/USD",
-    "spot://WBTC/USD",
     "spot://WETH/USD",
```

Comment:
oh, I think matic should be here, and in assets it should be with zero address

Reply:
be5bb6a0fe67790fb92eec93789c01a6e45e7ceb